WebViewControl/WebView.JavascriptExecutor.cs

```diff
@@ -272,6 +272,7 @@ internal class JavascriptExecutor : IDisposable {
             }
 
             private static Exception ParseException(Exception exception, IEnumerable<string> evaluatedScriptFunctions) {
+                var evaluatedScriptsMessage = " Evaluated Scripts: " + evaluatedScriptFunctions.Count() + ";";
```

Comment:
whats this info used for?
i dont think this is a good idea... we're injecting extra stuff in the message that the developers didn't specified...

Reply:
In the context of this issue, I prepared [this](https://docs.google.com/presentation/d/11axppbIUb6cNywp73foZEKj7skERAu00-czsBc25h1U/edit) report, where you can find our main conclusions during the investigation related with this task.
This modification will provide us the number of script that were executed by the `BulkExecuteScripts`, since we suspect that having multiple requests might be related with the timeou.